src/change/sdp.rs

```diff
@@ -193,33 +193,50 @@ impl<'a> SdpApi<'a> {
     /// The mid is not valid to use until the SDP offer-answer dance is complete and
     /// the mid been advertised via [`Event::MediaAdded`][crate::Event::MediaAdded].
     ///
+    /// * `stream_id` is used to synchronize media. It is `a=msid-semantic: WMS <streamId>` line in SDP.
+    /// * `track_id` is becomes both the track id in `a=msid <streamId> <trackId>` as well as the
+    ///   CNAME in the RTP SDES.
+    ///
     /// ```
     /// # use str0m::{Rtc, media::MediaKind, media::Direction};
     /// let mut rtc = Rtc::new();
     ///
     /// let mut changes = rtc.sdp_api();
     ///
-    /// let mid = changes.add_media(MediaKind::Audio, Direction::SendRecv, None);
+    /// let mid = changes.add_media(MediaKind::Audio, Direction::SendRecv, None, None);
     /// ```
-    pub fn add_media(&mut self, kind: MediaKind, dir: Direction, cname: Option<String>) -> Mid {
+    pub fn add_media(
+        &mut self,
+        kind: MediaKind,
+        dir: Direction,
+        stream_id: Option<String>,
```

Comment:
Note: Per the spec a given track can belong to many streams, not just the one and in the spec API you can add new streams after the initial negotiation. Not something we have to necessarily address, we can limit ourselves to one stream, but I thought I'd mention it.

Some relevant discussions:

* https://bugs.chromium.org/p/chromium/issues/detail?id=1342010
* https://github.com/w3c/webrtc-pc/issues/2782